PinClustering/PinClustering/ViewControllers/MapViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  MapViewController.swift
+//  PinClustering
+//
+//  Created by Norbert Agoston on 06/09/16.
+//  Copyright © 2016 Norbert Agoston. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class MapViewController: UIViewController {
+
+    @IBOutlet weak var map: MKMapView!
+    
+    var pointsOfInterest = [AnyObject]()
+    
+    var town: Town? {
+        didSet {
+            loadLocations()
+        }
+    }
+    
+    // MARK: Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.navigationBarHidden = false
+    }
+    
+    // MARK: Private
+    
+    func loadLocations() {
+        guard let url = town?.webServiceURL else {
+            return
+        }
+
+        NetworkManager.sharedInstance.httpGet(url) { (locations, error) in
+            guard let actualLocations = locations else {
```

Comment:
you don't really need to rename the variable. You could use guard let locations = locations else {}